momaland/envs/ingenious/ingenious.py

```diff
@@ -0,0 +1,238 @@
+"""Multi-objective Ingenious environment for MOMAland.
+
+This environment is based on the Ingenious game: https://boardgamegeek.com/boardgame/9674/ingenious
+Every color is a different objective. The goal in the original game is to maximize the minimum score over all colors,
+however we leave the utility wrapper up to the users and only return the vectorial score on each color dimension.
+"""
+
+import functools
+import random
+
+# from gymnasium.utils import EzPickle
+from typing_extensions import override
+
+import numpy as np
+from gymnasium.logger import warn
+from gymnasium.spaces import Box, Dict, Discrete
+from pettingzoo.utils import wrappers
+
+from momaland.envs.ingenious.ingenious_base import ALL_COLORS, IngeniousBase
+from momaland.utils.env import MOAECEnv
+
+
+def env(**kwargs):
+    """Autowrapper for the item gathering problem.
```

Comment:
These seems some conservative code. I am not sure whether it could be changed. I find these code in other env and even some in pettingzoo document.

Reply:
I meant just the name in the comment; I assume this code was copied from the item gathering environment. So in your case, it would be """Autowrapper for the ingenious game.""" or something similar 😃